src/Analyzers/Core/Analyzers/Formatting/FormatterHelper.cs

```diff
@@ -30,10 +31,10 @@ internal static IEnumerable<AbstractFormattingRule> GetDefaultFormattingRules(IS
     /// <param name="cancellationToken">An optional cancellation token.</param>
     /// <returns>The formatted tree's root node.</returns>
     public static SyntaxNode Format(SyntaxNode node, ISyntaxFormatting syntaxFormattingService, SyntaxFormattingOptions options, CancellationToken cancellationToken)
-        => Format(node, [node.FullSpan], syntaxFormattingService, options, rules: null, cancellationToken: cancellationToken);
+        => Format(node, [node.FullSpan], syntaxFormattingService, options, rules: default, cancellationToken: cancellationToken);
```

Comment:
changed things to be explicitly ImmutableArrays, not IEnumerables.  So lots of fallout of that.